Task/MCU/Floor1_USART_inputdata/Core/Src/main.c

```diff
@@ -173,7 +173,6 @@ int main(void)
       //set the time
       setInitialTime();
   }
-
   setAlarm();
 #endif
```

Comment:
Add comment after #endif so that we can identify easily

Reply:
done